recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   sha256: bee1cbd4c499ef3ed020f9ce152b722b8e30e69a8b25abaa116673ebda833875
 
 build:
-  number: 0
```

Comment:
I don't think we need to re-upload, because the label was already removed.

Reply:
I agree, but this might have created some inconsistency with the metadata (this definitely is the case for the one inside of the conda packages themselves because the channel isn't the proper one).

My assumption is that the cost of a reupload is cheap and is worth for consistency.

What do you think?

Reply:
ok